projects/invisible-island.net/ncurses/package.yml

```diff
@@ -31,6 +31,13 @@ build:
     rmdir ncursesw
     ln -sf . ncursesw
 
+    # fix hardcoded paths
+    sed -i.bak 's|{{prefix}}|\${pcfiledir}/../..|g' {{prefix}}/lib/pkgconfig/*.pc
+    rm {{prefix}}/lib/pkgconfig/*.bak
+
+    sed -i.bak 's|{{prefix}}|\${NCURSES_ROOT}|g' {{prefix}}/bin/ncursesw{{version.major}}-config
```

Comment:
Instead creating a envvar, this should probably use the dirname dirname $0 logic we use in other places.

Reply:
Hmm. Yes it will be better. I'll do it now